src/main/java/com/epam/prd/jjop/helpers/objects/NullWonder.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.prd.jjop.helpers.objects;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Description of @see java.util.Objects regarding null problem solving
+ * <p>
+ * This class show examples of all methods to check for null.
+ * Description of the differences between these methods can be found here.
+ * </p>
+ *
+ * @author Krzysztof Dolański
+ * Objects class form java.util @since 1.7
```

Comment:
Probably you mean "from" :D I would move this to the first sentence or drop entirely, to be honest.